File: tourneyClient/src/models/entities/Characters/Steve.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Steve: Character =
{
    id: CharacterId.Steve,
    characterName: CharacterName.STEVE,
    archetype: Archetype.TRAPPER,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.MIDDLEWEIGHT,
    tierPlacement: TierPlacement.S
}

export default Steve; 

File: tourneyAPI/Models/Entities/Characters/Steve.cs
namespace Entities;

using Enums;


// Defines the competitive profile metadata for this playable character.
public class Steve : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Steve()
    {
        Id = CharacterId.Steve;
        characterName = CharacterName.STEVE;
        archetype = Archetype.TRAPPER;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.MIDDLEWEIGHT;
        tierPlacement = TierPlacement.S;
    }

}